pytorch_memlab/utils.py

```diff
@@ -1,5 +1,5 @@
 from calmsize import size as calmsize
-
+from math import isnan
 
 def readable_size(num_bytes):
-    return '{:.2f}'.format(calmsize(num_bytes))
+    return '' if isnan(num_bytes) else '{:.2f}'.format(calmsize(num_bytes))
```

Comment:
I think it's ok to print NaN directly.

Reply:
pandas uses `nan` as a filler value to indicate missing values; the HTML output is pretty ugly without this - all the empty lines with no reported data get nans.